express-notes-api/index.js

```diff
@@ -0,0 +1,148 @@
+const express = require('express');
+const fs = require('fs');
+const app = express();
+const data = require('./data.json');
+const notes = data.notes;
+const notesArray = [];
+const nextId = data.nextId;
+const resError = {};
+
+for (const key in notes) {
+  notesArray.push(notes[key]);
+}
+
+app.get('/api/notes', (req, res) => {
+  res.send(notesArray);
+});
```

Comment:
I recommend creating a brand new array for each `GET /api/notes`. Otherwise, you have to mess with this array every time the notes change which is a pain. Imagine `POST` or `DELETE` followed by a `GET /api/notes`. The array will be out of date.